Galaxus.Functional/(Either)/Either2.cs

```diff
@@ -67,6 +67,16 @@ public Either(B b)
         /// </summary>
         public Discriminant2 Discriminant { get; }
 
+        /// <summary>
+        /// Indicates if field A is in use.
```

Comment:
I would remove the _if_ or replace it with _that_.
https://english.stackexchange.com/questions/68379/which-is-correct-indicates-or-indicates-that

Reply:
This comment refers to 5 occurrences of the same spelling issue.